diff --git a/node_modules/@leanprover/infoview/dist/index.d.ts b/node_modules/@leanprover/infoview/dist/index.d.ts
index 27b2c72..abb55de 100644
--- a/node_modules/@leanprover/infoview/dist/index.d.ts
+++ b/node_modules/@leanprover/infoview/dist/index.d.ts
@@ -6,7 +6,7 @@ export { EditorContext, VersionContext } from './infoview/contexts';
 export { EditorConnection } from './infoview/editorConnection';
 export { RpcContext } from './infoview/rpcSessions';
 export { ServerVersion } from './infoview/serverVersion';
-export { GoalLocation, GoalsLocation } from './infoview/goalLocation';
+export { LocationsContext, GoalLocation, GoalsLocation } from './infoview/goalLocation';
 export { importWidgetModule, DynamicComponent, DynamicComponentProps, PanelWidgetProps } from './infoview/userWidget';
 export { InteractiveCode, InteractiveCodeProps } from './infoview/interactiveCode';
 export { renderInfoview } from './infoview/main';
